stacks/LinkdexStack.ts

```diff
@@ -1,14 +1,21 @@
 import { StackContext, Api, Bucket } from "@serverless-stack/resources"
 import { aws_s3 as s3 } from 'aws-cdk-lib'
 
-export function LinkdexStack({ app,stack }: StackContext) {
+export function LinkdexStack({ app, stack }: StackContext) {
   // either import an existing bucket or create a new one for dev.
   // see: https://docs.sst.dev/advanced/importing-resources
   const bucket = (process.env.BUCKET_NAME)
     ? new Bucket(stack, 'existing-cars', { cdk: { bucket: s3.Bucket.fromBucketName(stack, 'imported-cars', process.env.BUCKET_NAME) }})
     : new Bucket(stack, 'cars')
 
+  const zone = 'linkdex.dag.haus'
```

Comment:
```suggestion
  const zone = process.env.DOMAIN || 'linkdex.dag.haus'
```

Reply:
Good point. I think this needs to be tweaked so that we only set the `customDomain` config on the api if we find a env var, so that it can be run on any aws account without having to update config.